delfin/exporter/prometheus/exporter_server.py

```diff
@@ -26,19 +30,31 @@
                help='The exporter server host  ip'),
     cfg.IntOpt('metric_server_port', default=8195,
                help='The exporter server port'),
-    cfg.StrOpt('metrics_cache_file', default='/var/lib/delfin/delfin_exporter'
-                                             '.txt',
-               help='The temp cache file used for persisting metrics'),
+    cfg.StrOpt('metrics_dir', default='/var/lib/delfin/metrics',
+
+               help='The temp directory to keep incoming metrics'),
 ]
 cfg.CONF.register_opts(prometheus_opts, group=grp)
 cfg.CONF(sys.argv[1:])
 
 
 @app.route("/metrics", methods=['GET'])
 def getfile():
-    with open(cfg.CONF.PROMETHEUS_EXPORTER.metrics_cache_file, "r+") as f:
-        data = f.read()
-        f.truncate(0)
+    try:
+        os.chdir(cfg.CONF.PROMETHEUS_EXPORTER.metrics_dir)
+    except OSError as e:
+        LOG.error('Error opening metrics folder')
+        raise Exception(e)
+    file_list = []
+    for file in glob.glob("*.prom"):
+        file_list.append(file)
+    data = ''
+    for file in file_list:
```

Comment:
this loop can be merged into loop at 49, right?

Reply:
done